### PR TITLE
Fix imports.

### PR DESCRIPTION
**Pull request by conniey**
Fix checkstyle errors failing CI build.

### CODE REVIEW

File: sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/DirectorySample.java [hunk 1]
```diff
@@ -4,12 +4,9 @@
 
 import com.azure.core.http.rest.Response;
 import com.azure.core.util.configuration.ConfigurationManager;
-import com.azure.storage.file.DirectoryClient;
```
Comment (sima-zhu) [inline sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/DirectorySample.java]: I am curious why these import useless. The DirectoryClient is still in use.

Comment (alzimmermsft) [reply sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/DirectorySample.java]: It's because both files are in the same package.

Comment (conniey) [reply sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/DirectorySample.java]: Classes that live in the same package do not have to be imported. They are done automatically. This file did not live in the same package name before.

Comment (sima-zhu) [reply sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/DirectorySample.java]: Thanks! What's the reason of making the sample under the same package?

Comment (conniey) [reply sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/DirectorySample.java]: Jonathan mentioned that samples should live in the root package (ie. com.azure.messaging.eventhubs) for the respective client library.
https://azure.github.io/azure-sdk/java_documentation.html

**Pull request merged**